### PR TITLE
SRP6 fix

### PR DESCRIPTION
**Pull request by ahoiahoi**
this should fix one of the login issues
there is still an issue on the STS server when trying to re-login with a already authenticated session

### CODE REVIEW

File: Source/NexusForever.Shared/Cryptography/Srp6Provider.cs [hunk 1]
```diff
@@ -160,7 +160,11 @@ private static BigInteger ShaInterleave(BigInteger key)
                 .Reverse()
                 .ToArray();
 
-            int length = Math.Max(4, Array.IndexOf<byte>(T, 0) + 1);
+            int first0 = Array.IndexOf<byte>(T.Reverse().ToArray(), 0);
```
Comment (Therzok) [inline Source/NexusForever.Shared/Cryptography/Srp6Provider.cs]: Why double reverse?

Comment (ahoiahoi) [reply Source/NexusForever.Shared/Cryptography/Srp6Provider.cs]: to not alter the code any more than needed

**Comment by ahoiahoi** [comment]
Reopen

**Comment by ahoiahoi** [comment]
reopened after takedown

**Pull request merged**